### ISSUE TITLE
Improve post condition for no transfer guarantee

### ISSUE THREAD

**Issue by markmhendrickson** [leather-io/extension#1298]
This unit seems funky to me in general given that it's enforcing 0 STX (whereas it makes more sense when the value is positive):

<img width="469" alt="Screen Shot 2021-06-14 at 12 14 13" src="https://user-images.githubusercontent.com/28991/122049296-8e61a900-cde2-11eb-9fa6-91bb5e936ade.png">

@jasperjansz could we simplify for these cases with 0 asset transfers a la "The contract will not transfer any STX or the transaction will abort."

**Comment by jasperjansz** [comment]
I agree this is funky. If an app wants a function call to make no transfers, they don't need to add any post-conditions — just have them be evaluated in deny mode. That said, I agree we can make a better message if they do add a post-condition like this.

**Comment by markmhendrickson** [comment]
cc @radicleart for his take on removing post conditions and relying on deny mode in this case, since the example above came from Risidio / thisisnumberone

**Comment by mijoco-btc** [comment]
Hi - thanks - how do I set deny mode from the client - having trouble finding documentation - or is it enough to just run without the P.C.?

**Comment by markmhendrickson** [comment]
cc @pgray-hiro per the docs question here and whether we might want to add more guidance on how to go about this ☝️

**Comment by pgray-hiro** [comment]
Captured in this issue - https://github.com/blockstack/docs/issues/1202
